### ISSUE TITLE
Article Translations

### ISSUE THREAD

**Issue by ldoliv** [FLEXIcontent/flexicontent-cck#292]
Hello, I'm using FLEXIcontent 3.0.8.

Currently Flexicontent is allowing to translate an article that has already been translated. In my view this souldn't happen.

I end up with the original article, +2 duplicate content articles translated to the language i choose.
The orignal article gets associated to the last translated item or vice versa.

![image](https://cloud.githubusercontent.com/assets/6535818/9937577/45177dce-5d59-11e5-8d4b-71710a476419.png)

**Comment by ggppdk** [comment]
Hello

yes it does allow you to re-select an already used language
- the reason is because the batch task in items manager, allows you to select multiple items
- and because the items manager lists the associations, so you can see items already having this language

it is similar in item form, if you set a new association the old one is lost
i am not sure if we would want to force disallowing this

**Comment by ldoliv** [comment]
Ok, then maybe restrict it in the batch/translate process. :)

**Comment by ggppdk** [comment]
Yes but the question is should we skip them during the creation of new items ? despite the user having selected these

**Comment by ldoliv** [comment]
In my opinion yes, if you are translating and they already have a translation association with another article then ignore.

Allow in "batch" button, but ignore in "translate" button.

**Comment by ggppdk** [comment]
I changed title of this issue to:
Translation task in items manager does not prevent/protect the replacemnet of existing associations

and marked it as a low priority issue for v3.0.x
